.editorconfig

```diff
@@ -0,0 +1,25 @@
+# https://EditorConfig.org
+
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+max_line_length = 120
+block_comment_start = /*
+block_comment = *
+block_comment_end = */
+
+[*.md]
+max_line_length = 300
+trim_trailing_whitespace = false
+
+[.github/workflows/back-end.yml]
+max_line_length = 165
+
+[*.php]
+max_line_length = 300
```

Comment:
Please consider writing shorter lines ~120

Reply:
Haha, will do! It's been a while since I've had that as a standard as I'd like to read as much as possible in my viewport. ;)